index.html

```diff
@@ -40,11 +39,163 @@ <h3 class = "getintouch">
+      <li class="lan"> CSS</li>
+      <li class="lan">Javascript</li>
+    </ul>
+    <div class="btn-div">
+      <button class="btn" type = "submit"> See Project</button>
+    </div>
+  </div>
+  <div class="second-sec-div">
+    <div>
+      <img class="proj-img" src="images/Snapshoot Portfolio (3).png" alt="picture" >
+    </div>
+    <div>
+      <header class="proj-title">Multi-Post <br>
+        Stories</header>
+    </div>
+    <div class="proj-intro">
+      <span class="intro-1">CANOPY</span>
+      <img class="dot" src="images/Counter.png" alt="dot">
+      <span class="intro-2">Back end Dev</span>
+      <img class="dot" src="images/Counter.png" alt="dot">
+      <span class="intro-3">2015</span>
+    </div>
+    <p class="p-descrip">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="proj-lan">
+      <li class="lan">HTML</li>
+      <li class="lan"> CSS</li>
+      <li class="lan">Javascript</li>
+    </ul>
+    <div class="btn-div">
+      <button class="btn" type = "submit"> See Project</button>
+    </div>
+  </div>
+  </div>
+</section>
+<section class="sec-3">
+  <div class="section3-div">
+  <div class="third-section" >
+    <div>
+    <h1 class="title">
+      About Myself
+    </h1>
+    <p class="intro">
+      I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <h3 class = "getintouch">
+      LET'S CONNECT
+    </h3>
+    </div>
+    <ul class="social-media-icon" >
+     <li><a class = "social-media" href="#"><img src="images/github.png" alt="github"></a></li>
+     <li> <a class = "social-media"href="#"><img src="./images/linkedin.png" alt="linkedin"></a> </li>
+     <li><a class = "social-media"href="#"><img src="images/anglelist.png" alt="anglelist"></a> </li>
+     <li><a class = "social-media" href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+     <li><a class = "social-media"href="#"><img src="images/medium.png"alt="medium"></a></li>
+ </ul>
+  </div>
+<div class="middle-sec">
+  <div class="btn-divb">
+    <button class= "btn" type="submit">Get my resume</button>
+  </div>
+  </div>
+      <ul class="chev">
+      <li class="p-lan">Languages </li> 
+      <li><i class="fa fa-chevron-down"></i></li>
+      </ul>
+    <ul class="img">
+     <li><img class="lan-g" src="images/Frame 9.png" alt="Javascript"></li>
+    <li><img class="lan-g" src="images/Frame 10.png" alt=" HTML"></li>
+    <li><img class="lan-g" src="images/Frame 11.png" alt="CSS"></li>
+    </ul>
+    <ul class="chev">
+    <li class="p-lan">Frameworks </li>
+    <li><i class="fa fa-chevron-right"></i></li>
+    </ul>
+    <hr class="line">
+    <ul class="chev">
+    <li class="p-lan">Skills</li>
+    <li><i class="fa fa-chevron-right"></i> </li>
+    </ul>
+    <hr class="line">
+</div>
```

Comment:
- As specified by the previous reviewer please make sure the technology categories ("Languages", "Frameworks" and "Skills") are all wrapped in a `ul` tag and each technology is in an `li` tag, this should contain not only the heading of the technology but other related contents as well (like the list of languages). ✔